package.json

```diff
@@ -33,7 +33,6 @@
     "istanbul": "^0.4.5",
     "mocha": "^5.2.0",
     "mocha-eslint": "^4.1.0",
-    "mocha-jshint": "^2.3.1",
```

Comment:
You need to remove the reference to this from tests as well

Reply:
Done. :-)